examples/styles/2-Card-Detailed.css

```diff
@@ -2,37 +2,16 @@
   box-sizing: border-box;
 }
 
-input,
-button {
+.AppWrapper input,
+.AppWrapper button {
+  all: unset;
   -webkit-appearance: none;
   -moz-appearance: none;
   appearance: none;
   outline: none;
   border-style: none;
 }
 
-html {
-  background-color: #6772e5;
-  font-size: 16px;
-  font-family: Roboto, Open Sans, Segoe UI, sans-serif;
-  font-weight: 500;
-  font-style: normal;
-  text-rendering: optimizeLegibility;
-  height: 100%;
-}
-
-body {
-  height: 100%;
-  margin: 0;
-}
-
-#root {
-  height: 100%;
-  display: flex;
-  align-items: center;
-  justify-content: center;
-}
-
```

Comment:
(question) I'm not familiar with this example - were these essentially unused? Seems like we're not replacing them in this changeset.

Reply:
These were used, but I didn't think these styles were worth fixing the problem of conflicting global styles contaminating other stories, which is what the old window.location.href reloading was for.

Before (ignore missing credit card field - unrelated):
![image](https://user-images.githubusercontent.com/93609068/168184154-df4dac7c-6670-4c54-b4d0-4095baa3f52d.png)

After:
![image](https://user-images.githubusercontent.com/93609068/168183982-953bea2e-7023-4b68-9632-82b95b845cec.png)